homeassistant/components/sensor/tibber.py

```diff
@@ -68,10 +72,23 @@ def __init__(self, tibber_home):
         """Get the latest data and updates the states."""
         now = dt_util.utcnow()
         if self._tibber_home.current_price_total and self._last_updated and \
-           dt_util.as_utc(dt_util.parse_datetime(self._last_updated)).hour\
-           == now.hour:
+           self._last_updated.hour == now.hour and self._newest_data_timestamp:
             return
 
+        @Throttle(MIN_TIME_BETWEEN_UPDATES)
+        async def _fetch_data():
```

Comment:
Expose this as normal member variable, So It don't create a function every 1min that maybe not used and need cleanup from garbridge collector.